kernel/src/sched.rs

```diff
@@ -994,6 +1007,17 @@ impl Kernel {
                                                     old_upcall.into_subscribe_success()
                                                 }
                                                 Err((new_upcall, err)) => {
+                                                    if config::CONFIG.trace_syscalls {
+                                                        if let Some(currently_allocated_grants_count) = process.grant_allocated_count() {
+                                                            if currently_allocated_grants_count > allocated_grants_count {
+                                                                debug! ("[{:?}] error driver {} allocated wrong grant for upcalls", process.processid(), driver_number);
```

Comment:
It definitely seems like a rustfmt bug that this passes formatting with such long line length.

Reply:
It is a bug, I tried formatting it in several ways, no luck, I formatted it manually.

Reply:
Yeah, I've noticed that too. https://github.com/WyliodrinEmbeddedIoT/tock/pull/7/commits/9eb3e2ffcfc433cacc17a13d4689df30a854109f seems like a reasonable formatting + makes the CI work...